examples/va/nfv/ovs-dpdk/edpm/openstackdataplanenodeset.yaml

```diff
@@ -0,0 +1,201 @@
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
+    edpm-compute-1:
+      ansible:
+        ansibleHost: 192.168.122.101 # CHANGEME
+      hostName: edpm-compute-1
+      networks:
+      - defaultRoute: true
+        fixedIP: 192.168.122.101 # CHANGEME
+        name: CtlPlane
+        subnetName: subnet1
+      - name: InternalApi
+        subnetName: subnet1
+      - name: Storage
+        subnetName: subnet1
+      - name: Tenant
+        subnetName: subnet1
+  networkAttachments:
+    - ctlplane
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    managementNetwork: ctlplane
+    ansible:
+      ansibleUser: cloud-admin
+      ansiblePort: 22
+      ansibleVars:
+        # CHANGEME -- see https://access.redhat.com/solutions/253273
+        # edpm_bootstrap_command: |
+        #       subscription-manager register --username <subscription_manager_username> --password <subscription_manager_password>
+        #       podman login -u <registry_username> -p <registry_password> registry.redhat.io
+        service_net_map:
+          nova_api_network: internal_api
+          nova_libvirt_network: internal_api
+        # CHANGEME
+        timesync_ntp_servers:
+          - hostname: clock.redhat.com
+        # CPU pinning settings
+         # edpm nfv ovs dpdk config
+         edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=64 iommu=pt intel_iommu=on tsx=off isolcpus=2-11,14-23"  # CHANGEME
+         edpm_tuned_profile: "cpu-partitioning-powersave"  # CHANGEME
+         edpm_tuned_isolated_cores: "2-11,14-23"  # CHANGEME
+         edpm_nova_libvirt_qemu_group: "hugetlbfs"
+         edpm_ovs_dpdk_pmd_core_list: "1,13,2,14,3,15"  # CHANGEME
+         edpm_ovs_dpdk_socket_memory: "4096"  # CHANGEME
+         edpm_ovs_dpdk_memory_channels: "4"  # CHANGEME
+         edpm_ovs_dpdk_vhost_postcopy_support: "true"  # CHANGEME
+         edpm_ovn_bridge_mappings: ['dpdk2:br-link2','dpdk1:br-link1']  # CHANGEME
+        # edpm_network_config
+        # These vars are edpm_network_config role vars
+        edpm_network_config_hide_sensitive_logs: false
+        # CHANGEME (network_config[0].members[0].name) ---v
+        # CHANGEME (network_config[1].members[0].name) ---v
+        edpm_network_config_template: |
+          ---
+          {% set mtu_list = [ctlplane_mtu] %}
+          {% for network in role_networks %}
+          {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+          {%- endfor %}
+          {% set min_viable_mtu = mtu_list | max %}
+          network_config:
+          - type: ovs_bridge
+            name: {{ neutron_physical_bridge_name }}
+            mtu: {{ min_viable_mtu }}
+            use_dhcp: false
+            dns_servers: {{ ctlplane_dns_nameservers }}
+            domain: {{ dns_search_domains }}
+            addresses:
+            - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_subnet_cidr }}
```

Comment:
This is also out  of date. Please update so you are in-line with.

https://github.com/Jaganathancse/architecture/commit/5a8b7d665b981b1483452f906cd2464f04dec0b8